docs/source/configuration/configuration-run.rst

```diff
@@ -185,7 +185,7 @@ GraphStorm provides a set of parameters to control training hyper-parameters.
 
     - Yaml: ``sparse_optimizer_lr: 0.5``
     - Argument: ``--sparse-optimizer-lr 0.5``
-    - Default value: Same as lr.
```

Comment:
Why change the default value from lr to 0.01?

Reply:
revert to use lr for the sparse_optimizer_lr. thanks